blockchain/blockchain-wallet-v4-frontend
# fix(simple buy): use maxPossibleOrder instead of account balance for sb FUNDS orders

## Description (optional)
When a user would click the max buy on a simple buy order from their FUNDS the form would use the max available funds in the account instead of using the `maxPossibleOrder` limit we get from the /trades/limits endpoint

## Testing Steps (optional)
- put funds in your Gold account
- open simple buy and click max buy, if your funds are less than 25,000 it should put in your total available funds, if they are over 25,000 it should input 20,000 (max limit per transaction)

Test scenarios still needed
- [x] funds under 20k
- [x] silver tier user
- [x] different payment methods
